### ISSUE TITLE
Acceptance Tests fail 

### ISSUE THREAD

**Issue by northborn** [yapplabs/ember-tether#18]
Because tether appends elements to the body and thus outside the scope of the test container, I'm running into issues where using  `find`, `findWithAssert`, or `click` on an element inside an ember-tether component cannot find the element. I wrote this custom async test helper modeled after the actual Ember `click` helper except using jquery to grap the element rather than a `findWithAssert`, which correclty finds the element, but something gets lost in the transition and the tests still fail. 

```
import Ember from 'ember';

export default Ember.Test.registerAsyncHelper('unscopedClick', function(app, selector, context) {
  var $el = $(selector, context);
  Ember.run($el, 'mousedown');

  focus($el);

  Ember.run($el, 'mouseup');
  Ember.run($el, 'click');

  return app.testHelpers.wait();
});
```

Either the clicks still don't actually happen in the runloop, or the click is not truly async and doesn't wait at the end like it's supposed to. There must be a better way to handle acceptance tests for elements appended to the body.

**Comment by tomoguisuru** [comment]
+1

**Comment by chrislopresto** [comment]
@northborn @tomoguisuru - I just published version 0.3.1 which provides a reasonable™️ workaround possible in the latest Hubspot Tether release. Hopefully this does the trick for everyone.

### Details

Hubspot Tether works by appending tethered elements to the `<body>` tag. Unfortunately, this moves your
content outside of the Ember application `rootElement` during acceptance testing. This breaks event
dispatch and action handling, including traditional Ember test helpers like `click`.

In order to [short-circuit Hubspot's positioning behavior](https://github.com/HubSpot/tether/pull/98/), we must use static positioning on the
`#ember-testing-container` div as follows:

##### tests/index.html
```html
<style>
  #ember-testing-container {
    /* Set position static to short-circuit Hubspot Tether's positioning */
    /* https://github.com/HubSpot/tether/pull/98/ */
    position: static !important;
  }
</style>
```
